themes/openy_themes/openy_lily/sass/modules/_branch.scss

```diff
@@ -1082,3 +1082,66 @@ html.js {
     margin-right: 0.25em;
   }
 }
+
+.marker_tooltip {
```

Comment:
no need to add code to every theme, use global include https://github.com/ymcatwincities/openy/blob/8.x-1.13-dev/themes/openy_themes/scss/modules.scss to include your scss/sass file from all themes available in a system, - just recompile them all in case if there is a change

See https://github.com/ymcatwincities/openy/blob/8.x-1.13-dev/themes/openy_themes/openy_lily/sass/style.scss#L95 - the file already included in lily and rose